src/agent/adu_core_interface/inc/aduc/adu_core_interface.h

```diff
@@ -63,11 +64,7 @@ void AzureDeviceUpdateCoreInterface_Destroy(void** componentContext);
  * @brief A callback for an 'azureDeviceUpdateAgent' component's property update events.
  */
 void AzureDeviceUpdateCoreInterface_PropertyUpdateCallback(
-    IOTHUB_DEVICE_CLIENT_LL_HANDLE deviceClient,
-    const char* propertyName,
-    JSON_Value* propertyValue,
-    int version,
-    void* context);
+    ADUC_ClientHandle deviceClient, const char* propertyName, JSON_Value* propertyValue, int version, void* context);
```

Comment:
>deviceClient, [](start = 22, length = 13)

Probably a good time to rename this to something more generic, 'deviceUpdateClient' ? or 'clientHandle'?

Reply:
i like the latter to match ClientHandle

---
In reply to: [591972223](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591972223) [](ancestors = 591972223)

Reply:
Crap. There's probably a bunch of these. 

I will look and change throughout.

---
In reply to: [591974756](https://github.com/Azure/iot-hub-device-update/pull/12#discussion_r591974756) [](ancestors = 591974756,591972223)